README.md

```diff
@@ -23,8 +23,8 @@ jobs:
 
 ### Required
 
-- `devcard_id`: this is the unique hash of the devcard, it can be found in the URL of the devcard.
-  - e.g. `https://api.daily.dev/devcards/0b156485612243bfa39092.3.171e276.png` where the devcard_id is `0b156485612243bfa39092.3.171e276`
+- `devcard_id`: this is the unique id of the devcard, it can be found in the URL of the devcard.
+  - e.g. `https://api.daily.dev/devcards/v2/0b156485612243bfa39092.3.171e276.png` where the devcard_id is `0b156485612243bfa39092.3.171e276`
   - Can be found at [https://app.daily.dev/devcard](https://app.daily.dev/devcard)
```

Comment:
```suggestion
  - Can be found at [https://app.daily.dev/api/id](https://app.daily.dev/api/id)
```

Since the ID is the userID (I presume), we can just link to the ID page in the API?

Reply:
I'd rather link to the page with the settings, as we did before. Otherwise it might be confusing to the user.